src/util.js

```diff
@@ -139,6 +154,13 @@ function makeTitle(string) {
     return res;
 }
 
+/**
+ * Used to get the time of midnight tomorrow
+ * @returns {number} The unix time in milliseconds at midnight tomorrow.
+ */
+function getTomorrow() {
+    return new Date().setUTCHours(0, 0, 0, 0) + 8.64e7;
```

Comment:
Interesting way, but I'll allow it

Reply:
How would you do it?

Reply:
According to stackoverflow set the hours to 24 to get the next midnight and remove the `+ 8.64e7`